avalon/nuke/pipeline.py

```diff
@@ -171,13 +196,14 @@ def ls():
     See the `container.json` schema for details on how it should look,
     and the Maya equivalent, which is in `avalon.maya.pipeline`
     """
-    all_nodes = nuke.allNodes(recurseGroups=True)
+    all_nodes = nuke.allNodes(recurseGroups=False)
 
     # TODO: add readgeo, readcamera, readimage
```

Comment:
This todo seems to have been resolved since it now includes literally `all_nodes`?
Which basically means the `nodes` list comprehension does nothing except making a copy of the full list. As such, it's redundant?